kuka_kl_support/urdf/kl100_macro.xacro

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+  <xacro:macro name="kl100_links" params="">
+    <!-- links -->
+    <link name="rail_base_link">
+      <visual>
+        <geometry>
+          <mesh filename="package://kuka_kl_support/meshes/kl100/visual/rail_base_link.dae"/>
+        </geometry>
+      </visual>
+      <collision>
+        <geometry>
+          <mesh filename="package://kuka_kl_support/meshes/kl100/collision/rail_base_link.stl"/>
+        </geometry>
+      </collision>
+    </link>
+    <link name="rail_moving_link">
+      <visual>
+        <geometry>
+          <mesh filename="package://kuka_kl_support/meshes/kl100/visual/rail_moving_link.dae"/>
+        </geometry>
+      </visual>
+      <collision>
+        <geometry>
+          <mesh filename="package://kuka_kl_support/meshes/kl100/collision/rail_moving_link.stl"/>
+        </geometry>
+      </collision>
+    </link>
+  </xacro:macro>
+  <xacro:macro name="kl100_joints" params="robot_base_link *origin">
+    <!-- joints -->
+    <joint name="world-rail_base" type="fixed">
+      <parent link="world"/>
+      <child link="rail_base_link"/>
+      <xacro:insert_block name="origin"/>
+    </joint>
+    <joint name="rail_joint" type="prismatic">
+      <parent link="rail_base_link"/>
+      <child link="rail_moving_link"/>
+      <origin rpy="0.0 0.0 0.0" xyz="0.0 0.0 0.0"/>
+      <axis xyz="1 0 0"/>
+      <limit lower="-1" upper="1" effort="100.0" velocity="1.0"/>
```

Comment:
I assume the effort/velocity values here are again dummys, indicate with a TODO comment